crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -600,7 +602,7 @@ mod tests {
 
     /// Create a data transfer component with a target object store that is deleted once the test is finished.
     async fn create_data_transfer_component(
-        metadata_manager: Arc<MetadataManager>,
+        metadata_manager: Arc<TableMetadataManager<Sqlite>>,
```

Comment:
Maybe renamed to `table_metadata_manager` for consistency.

Reply:
Fixed.